docs/netlify.toml

```diff
@@ -39,6 +39,31 @@
   to = "/"
   status = 301
 
+[[redirects]]
+  # redirect cloud orgs admin page from user manual
+  from = "/user-guide/cloud/609601/org-administration"
+  to = "/manuals/administrator/cloud/609601/organizations"
+  status = 301
+
+[[redirects]]
+  # redirect cloud roles and permissions page from user manual
+  from = "/user-guide/cloud/968372/roles-permissions"
+  to = "/manuals/administrator/cloud/968372/roles-permissions"
+  status = 301
+
+
+[[redirects]]
+  # redirect cloud orgs admin page from user manual
+  from = "/user-guide/cloud/609601/org-administration"
+  to = "/manuals/administrator/cloud/609601/organizations"
+  status = 301
+
+[[redirects]]
+  # redirect CI pages from user manual
```

Comment:
@d3rp3tt3 @mircubed this will affect some of the links on https://dagger.io/integrations. Redirections are in place in this PR but it would better to update the links anyway to point to the new pages.